sdk/src/Services/DynamoDBv2/Custom/Conversion/SchemaV1.cs

```diff
@@ -252,11 +252,11 @@ protected override bool TryTo(DateTime value, out Primitive p)
         }
         protected override bool TryFrom(Primitive p, Type targetType, out DateTime result)
         {
-            if (DateTime.TryParseExact(p.StringValue, AWSSDKUtils.ISO8601DateFormat, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal, out result))
+            if (DateTime.TryParseExact(p.StringValue, AWSSDKUtils.ISO8601DateFormat, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal, out result))
```

Comment:
I know this is called out as a breaking change, but if a user was previously using this method and passing in a time zone identifier like "PST", with this change the outputted DateTime would be converted to UTC which is a sneaky runtime breaking change right? Could a customer unknowingly start storing Datetimes as utc when they were storing them as local time?

Or are we basically taking the stance that time should be stored as UTC and that this previous behavior was just incorrect?

Reply:
It is a runtime breaking change. Norm and I discussed this and decided to return UTC by default. The good news is the datetimes are always stored as UTC in the database now. It it is UTC, LocalTime, or Unspecified it will convert `ToUniversal` before storing it in the DB so round tripping is fine.

The gotcha is if the user gets the UTC value back and does a datetime.Year or datetime.ToString("yyyy-mm-dd hh:MM;ss") for example where DateTime simply fills in the values. It will not use the Kind as part of the value fill. So the user could be formatting the string thinking it is local time still and it is actually the UTC values now.